db/migrate/20170601233834_delete_available_column_from_game.rb

```diff
@@ -0,0 +1,4 @@
+class DeleteAvailableColumnFromGame < ActiveRecord::Migration[5.0]
+  def change
+  end
```

Comment:
This is empty!

Instead of another migration. You could also use db:rollback and then delete the add_available migration.